test/test_live_coinc_compare.py

```diff
@@ -47,7 +47,9 @@ def get_trigs(self):
                     0,
                     self.num_templates,
                     size=self.num_trigs
-                ).astype(np.int32)
+                ).astype(np.int32),
+                "mass1": np.random.uniform(2.0, 100.0, size=self.num_trigs).astype(np.float32),
+                "mass2": np.random.uniform(2.0, 100.0, size=self.num_trigs).astype(np.float32)
```

Comment:
I think `uniform()` takes a `dtype` kwarg, which makes the code shorter, clearer and slightly more efficient.

Reply:
I did follow the example set by the other parameters above, looks like it doesn't have `dtype` which is odd:

Code:
`"mass1": np.random.uniform(2.0, 100.0, size=self.num_trigs, dtype=np.float32)`

Error:
`TypeError: uniform() got an unexpected keyword argument 'dtype'`

Reply:
Bummer, that is unexpected. Nevermind then.